src/ConnectedMode/Binding/IBindingProcess.cs

```diff
@@ -38,8 +38,6 @@ internal interface IBindingProcess
 
         Task<bool> SaveServerExclusionsAsync(CancellationToken cancellationToken);
 
-        void SaveRuleConfiguration(CancellationToken cancellationToken);
```

Comment:
Remove redundant method from the interface, implementing classes and tests.

No other product changes.